bindings/yarp.i

```diff
@@ -597,6 +600,15 @@ typedef yarp::os::BufferedPort<Vector> BufferedPortVector;
 %template(TypedReaderCallbackVector) yarp::os::TypedReaderCallback<yarp::sig::VectorOf<double> >;
 %template(BufferedPortVector) yarp::os::BufferedPort<yarp::sig::VectorOf<double> >;
 
+%template(VectorInt) yarp::sig::VectorOf<int>;
+#if SWIG_VERSION < 0x030012
```

Comment:
Probably copied from somewhere else, but we now depend on swig `3.0.12`, therefore this `if` could be removed

Reply:
Yes, it has been copied from the `VectorOf<double>` counterpart. I will remove the `if` then.